src/data/taskList.ts

```diff
@@ -0,0 +1,72 @@
+import { SystemIntakeForm } from 'types/systemIntake';
+import { BusinessCaseModel } from 'types/businessCase';
+import { businessCaseInitialData } from 'data/businessCase';
+
+export const calculateIntakeStatus = (intake: SystemIntakeForm) => {
+  if (intake.id === '') {
+    return 'START';
+  }
+  if (intake.status === 'DRAFT') {
+    return 'CONTINUE';
+  }
+  return 'COMPLETED';
+};
+
+export const chooseIntakeLink = (intake: SystemIntakeForm, status: string) => {
+  const newIntakeLink = '/system/new';
+  if (intake.id === '') {
+    return newIntakeLink;
+  }
+  let link: string;
```

Comment:
You're mixing two approaches here, one with an early return, and one with assigning a variable and then returning it at the end of the function. I would probably choose one approach or the other, rather than mix them.